.pre-commit-config.yaml

```diff
@@ -3,3 +3,11 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
+  - repo: local
+    hooks:
+      - id: jupyter-nb-clear-output
+        name: jupyter-nb-clear-output
```

Comment:
name should be different from id and describe what the hook does
For example: `Clear jupyter notebooks output and metadata`